src/components/rules-creation-page/steps/SchedulerStep.jsx

```diff
@@ -56,7 +57,11 @@ const SchedulerStep = ({ syncRule, onChange }) => {
             value: selectedCron.value || "",
             component: DropDown,
             props: {
-                hintText: i18n.t("Select frequency"),
+                hintText: i18n.t(
+                    isValidCronExpression(syncRule.frequency)
+                        ? cronstrue.toString(syncRule.frequency)
+                        : "Select frequency"
```

Comment:
I'd keep the `i18n.t` here.

Reply:
See https://github.com/EyeSeeTea/metadata-synchronization/pull/134/commits/d4eebaf7a6ac2897e547c59cf1986b291d022208